defs/lib.bzl

```diff
@@ -0,0 +1,31 @@
+
+load("@io_bazel_rules_closure//closure:defs.bzl", _closure_js="closure_js_library")
+
+DEFAULT_SUPPRESSIONS = [
+    "JSC_IMPLICITLY_NULLABLE_JSDOC",
+    "JSC_NULLABLE_RETURN_WITH_NAME",
+    "JSC_UNKNOWN_EXPR_TYPE"]
+
+
+def js_library(
+        name,
+        srcs = [],
+        convention = "GOOGLE",
+        deps = [],
+        exports = [],
+        suppress = []):
```

Comment:
all build rules move through here, which is nice, because we can apply stuff tree-wide